### PR TITLE
chore: cleanups in ipampoolmonitor

### PR DESCRIPTION
**Pull request by rbtr**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: cns/restserver/internalapi.go [hunk 1]
```diff
@@ -225,10 +225,7 @@ func (service *HTTPRestService) ReconcileNCState(
 	if returnCode != types.Success {
 		return returnCode
 	}
-	returnCode = service.UpdateIPAMPoolMonitorInternal(scalar, spec)
-	if returnCode != types.Success {
-		return returnCode
-	}
+	service.IPAMPoolMonitor.Update(scalar, spec)
```
Comment (thatmattlong) [inline cns/restserver/internalapi.go:228]: unchecked error?

File: cns/cnsclient/httpapi/client.go [hunk 2]
```diff
@@ -27,14 +27,8 @@ func (client *Client) CreateOrUpdateNC(ncRequest cns.CreateNetworkContainerReque
...
-func (client *Client) UpdateIPAMPoolMonitor(scalar nnc.Scaler, spec nnc.NodeNetworkConfigSpec) error {
-	returnCode := client.RestService.UpdateIPAMPoolMonitorInternal(scalar, spec)
-
-	if returnCode != 0 {
-		return fmt.Errorf("Failed to update IPAM pool monitor scalar: %+v, spec: %+v, errorCode: %d", scalar, spec, returnCode)
-	}
-
-	return nil
+func (client *Client) UpdateIPAMPoolMonitor(scalar nnc.Scaler, spec nnc.NodeNetworkConfigSpec) {
+	client.RestService.IPAMPoolMonitor.Update(scalar, spec)
```
Comment (thatmattlong) [inline cns/cnsclient/httpapi/client.go:31]: unchecked error?

File: cns/restserver/internalapi.go (hunk 1 above)
Comment (rbtr) [reply cns/restserver/internalapi.go:228]: `Update` can't return an error, that was the impetus of this simplification

File: cns/cnsclient/httpapi/client.go (hunk 2 above)
Comment (rbtr) [reply cns/cnsclient/httpapi/client.go:31]: SAA

**Review by thatmattlong — APPROVED** [review]

**Pull request merged**